### DISCUSSION TITLE
我想问一下手机怎么改音量?

### DISCUSSION THREAD

**Discussion by lsp-op** [AlistGo/alist#1108] [Q&A]
手机版怎么改音乐的音量？我看电脑版上面有，手机版没找到，
![Screenshot_2022-05-16-20-41-24-424_com android browser](https://user-images.githubusercontent.com/105648422/168594883-1b8e6676-39ac-4265-ad97-baf465b79c25.jpg)
更新完之后加载不开图片！在后台改过在2.5.1版本正常！
链接。
http://154.86.27.106:5244/%E9%9F%B3%E4%B9%90/%E9%99%88%E5%A5%95%E8%BF%85%20-%20%E5%AD%A4%E5%8B%87%E8%80%85.mp3

**Comment by lsp-op** [comment]
嗯，链接好像错了，点击直接下载。http://154.86.27.106:5244/%E9%9F%B3%E4%B9%90

**Comment by anwen-anyi** [comment]
啥？音量你减小手机音量不行么...ヾ(。￣□￣)ﾂ゜゜゜

**Comment by jxxcr** [comment]
你好，是如何在手机上访问的啊
